JavaScript/call-apply-bind.md

```diff
@@ -0,0 +1,148 @@
+# Call(), Apply() & Bind()
+___
+
+#### In essence, the purpose of the Call(), Apply(), Bind() methods is to enable the ability to control what the ```this``` variable ends up being when a function is run.
```

Comment:
Thanks very much, good suggestions. Definitely makes it a bit nicer to read : ) 

I have corrected and re-uploaded now.   Could you please have a look and let me know if you think this is ready to be merged to master branch?